jvb/rootfs/defaults/jvb.conf

```diff
@@ -0,0 +1,52 @@
+{{ $JVB_TCP_PORT := .Env.JVB_TCP_PORT | default "4443" }}
+{{ $JVB_TCP_MAPPED_PORT := .Env.JVB_TCP_MAPPED_PORT | default $JVB_TCP_PORT }}
+videobridge {
+    ice {
+        udp {
+            port = {{ .Env.JVB_PORT }}
+        }
+        tcp {
+            enabled = {{ not .Env.JVB_TCP_HARVESTER_DISABLED }}
+            port = {{ .Env.JVB_TCP_PORT }}
+{{ if not (eq $JVB_TCP_PORT $JVB_TCP_MAPPED_PORT) }}
+            mapped-port = {{ $JVB_TCP_MAPPED_PORT }}
+{{ end }}
+        }
+    }
+    apis {
+        xmpp-client {
+            configs {
+                shard {
+                    HOSTNAME = {{ .Env.XMPP_SERVER }}
+                    DOMAIN = {{ .Env.XMPP_AUTH_DOMAIN }}
+                    USERNAME = {{ .Env.JVB_AUTH_USER }}
+                    PASSWORD = {{ .Env.JVB_AUTH_PASSWORD }}
+                    MUC_JIDS = {{ .Env.JVB_BREWERY_MUC }}@{{ .Env.XMPP_INTERNAL_MUC_DOMAIN }}
+                    MUC_NICKNAME = {{ .Env.HOSTNAME }}
+                    DISABLE_CERTIFICATE_VERIFICATION = true
+                }
+            }
+        }
+    }
+    stats {
+        enabled = true
+        transports = [
+            {
+                type = "muc"
+                interval = 5 seconds
+            }
+        ]
+    }
+}
+
+ice4j {
+    harvest {
+        mapping {
+            stun {
+{{ if .Env.JVB_STUN_SERVERS }}
+                [ "{{ join "\",\"" (splitList "," .Env.JVB_STUN_SERVERS) }}" ]
```

Comment:
it complains about the syntax here.

```
Caused by: com.typesafe.config.ConfigException$Parse: /config/jvb.conf: 43: expecting a close parentheses ')' here, not: '['
```

Here is the generated section:

```
ice4j {
    harvest {
        mapping {
            stun {

                [ "meet-jit-si-turnrelay.jitsi.net:443" ]

            }   
        }   
    }   
}   
```

Reply:
🤦  Ah--the error is confusing but I see it now.  It's missing the field name before the array.